common-controller/internal/operator/controller/ratelimitpolicy_controller.go

```diff
@@ -136,14 +136,17 @@ func (ratelimitReconsiler *RateLimitPolicyReconciler) Reconcile(ctx context.Cont
 		return ctrl.Result{}, nil
 	}
 	logger.Info("add api ratelimit")
-	var vhost, resolveRatelimit = ratelimitReconsiler.marshelRateLimit(ctx, ratelimitKey, ratelimitPolicy)
+	var vhost, resolveRatelimit, customRateLimitPolicy = ratelimitReconsiler.marshelRateLimit(ctx, ratelimitKey, ratelimitPolicy)
 	logger.Info("resolveRatelimitxxxx", resolveRatelimit)
-	if vhost == nil {
-		logger.Info("vhost is nil hence API deleted")
-		return ctrl.Result{}, nil
-	}
+	logger.Info("resolveCustomRateLimitPolicy", customRateLimitPolicy)
+
+	// if vhost == nil {
```

Comment:
Any reason to remove this?

Reply:
we need to check vhost && !customRatelimit then continue, becasuse if we host is null and flow continue api ratelimit will braek

Reply:
fixed